std-bits/table/src/main/java/org/enso/table/data/column/builder/Builder.java

```diff
@@ -32,7 +32,8 @@ public interface Builder {
    * */
   int MAX_SIZE = Integer.MAX_VALUE;
 
-  private static int checkSize(long size) {
+  /** Checks that the size is within the maximum allowed. */
+  static int checkSize(long size) {
```

Comment:
Why no longer `private`? I don't see usages outside of this class (or am I missing something?). I think this is an implementation detail so I'd keep it private if possible.

Reply:
Using it to sanity check other places where things are being resized etc.

Reply:
Ah now I've found an usage.

For some reason the `github.dev` editor showed me only usages inside of `Builder`...  or at least how I saw it.